os/arch/arm/src/amebad/amebad_timerisr.c

```diff
@@ -144,9 +151,9 @@ void up_timer_initialize(void)
 
   putreg32((NVIC_SYSTICK_CTRL_CLKSOURCE | NVIC_SYSTICK_CTRL_TICKINT |
             NVIC_SYSTICK_CTRL_ENABLE), NVIC_SYSTICK_CTRL);
+  DiagPrintf("%08x %08x\r\n", getreg32(NVIC_SYSTICK_CTRL), getreg32(NVIC_SYSTICK_RELOAD));

```

Comment:
I think this should be changed to something dbg.

Reply:
fixed in 3e6e8b8